src/llmq/dkgsessionhandler.cpp

```diff
@@ -110,7 +110,7 @@ bool CDKGPendingMessages::HasSeen(const uint256& hash) const
 void CDKGPendingMessages::Misbehaving(const NodeId from, const int score, PeerManager& peerman)
 {
     if (from == -1) return;
-    peerman.Misbehaving(from, score);
+    peerman.UnitTestMisbehaving(from, score);
```

Comment:
25144: I don't like this approach, we need to find a better way

Reply:
Sure , I will try another approach